File: exercicio e-commerce/php/produto.php
<?php
class Produto {
    private $cod;
    private $desc;
    private $qtde;
    private $valor;

    public function cadastrar() {
        $this->cod = readline('Digite o codigo do produto: ');
        $this->desc = readline('Digite a descrição do produto: ');
        $this->qtde = readline('Digite a quantidade: ');
        $this->valor = readline('Digite o valor do produto: ');
    }

    public function getCod() {
        return $this->cod;
    }

    public function setCod($cod) {
        $this->cod = $cod;
    }

    public function getDesc() {
        return $this->desc;
    }

    public function setDesc($desc) {
        $this->desc = $desc;
    }

    public function getQtde() {
        return $this->qtde;
    }

    public function setQtde($qtde) {
        $this->qtde = $qtde;
    }

    public function getValor() {
        return $this->valor;
    }

    public function setValor($valor) {
        $this->valor = $valor;
    }
}
?>

File: exercicio e-commerce/python/produto.py
class Produto:
    def cadastrar(self):
        self.__cod = input('Digite o codigo do produto: ')
        self.__desc = input('Digite a descrição do produto: ')
        self.__qtde = input('Digite a quantidade: ')
        self.__valor = input('Digite o valor do produto: ')

    @property
    def cod(self):
        return self.__cod

    @cod.setter
    def cod(self, cod):
        self.__cod = cod

    @property
    def desc(self):
        return self.__desc

    @desc.setter
    def desc(self, desc):
        self.__desc = desc

    @property
    def qtde(self):
        return self.__qtde

    @qtde.setter
    def qtde(self, qtde):
        self.__qtde = qtde

    @property
    def valor(self):
        return self.__valor

    @valor.setter
    def valor(self, valor):
        self.__valor = valor
